### PR TITLE
Advance functionality

### PR DESCRIPTION
**Pull request by MMercer8989**
I have fixed previous issues with the Bracket.nextRound() method as well as problems associated with it. To correct the problems I have made alterations to Main.java, Bracket.java, Match.java, and Model.java. Each of these alterations tie together and work as I believe they were intended, more or less (this was difficult to determine sometimes as there was no clear documentation). The only noticeable change to the user would be the removal of the "ADVANCE" command as well as the removal of a parameter to "UPDATE-MATCH". these changes were made primarily to avoid user error which could effect nextRound().

### CODE REVIEW

**Comment by mikeknows** [comment]
Merge conflict after accepting logging class PR.



                    case "ADVANCE":
                    	log.writeActionlog("Command Entered: " +args.get(0));
                        Model.advanceTournament();
                        return;

**Comment by MMercer8989** [comment]
I have resolved the listed merge conflicts and re-merged. Sorry for just now getting to this, I didn't see your email until recently.

File: src/wrestlingtournamentcli/Main.java [hunk 1]
```diff
@@ -1,139 +1,142 @@
...
-                + "HELP //Display list of available commands\n"
-                + "LOAD TournamentName //Loads all files associated with this tournament name\n"
-                + "SAVE TournamentName //Saves all information associated with this tournament to files with tournamentName\n"
-                + "NAME TournamentName //Changes the tournament's name"
-                + "IMPORT-TEAMS FileName //Parses the provided file for Team objects\n"
-                + "IMPORT-WRESTLERS FileName //Parses the provided file for Wrestler objects\n"
-                + "VIEW-WRESTLER WrestlerName //Looks for the wrestler and prints his/her information\n"
-                + "UPDATE-MATCH matchNumber winningColor greenPoints redPoints fallType(int) fallTime\n");
-                }
-}
```
Comment (mikeknows) [inline src/wrestlingtournamentcli/Main.java:453]: I need to be able to easily see the changes made. It appears that you deleted the original Main.java, and copied in yours with the changes you made. Please stay in the IDE when making changes.

**Review by mikeknows — CHANGES_REQUESTED** [review]
Another conflict after accepting a PR

File: src/wrestlingtournamentcli/Main.java (hunk 1 above)
Comment (MMercer8989) [reply src/wrestlingtournamentcli/Main.java:453]: I'm not sure why it's indicating that main is causing a conflict, to my knowledge I hadn't made any changes of this scale; when I go to resolve conflicts both segments of code appear to be the same. Will you be having regular office hours this week?

**Comment by MMercer8989** [comment]
I have changed it this time by switching case 7 to be case 6, making sure to change any necessary pieces of code that this could effect within. This was the only real difference I noticed so I'm still not sure why the entire main class was causing a conflict.

**Comment by mikeknows** [comment]
Another merge conflict

**Pull request merged**